### PR TITLE
Updates to latest sdl version

### PR DESCRIPTION
**Pull request by comtom**
When I tried to run this project, couldn't make it run because some packages in sdl binding for go changed their names. These small changes fixes that :)

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func drawTitle(r *sdl.Renderer, text string) error {
 	defer f.Close()
 
 	c := sdl.Color{R: 255, G: 100, B: 0, A: 255}
-	s, err := f.RenderUTF8_Solid(text, c)
+	s, err := f.RenderUTF8Solid(text, c)
```
Comment (comtom) [inline main.go:86]: I forgot to add this to the description. This func changed its name.

File: scene.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func (s *scene) handleEvent(event sdl.Event) bool {
 		return true
 	case *sdl.MouseButtonEvent:
 		s.bird.jump()
-	case *sdl.MouseMotionEvent, *sdl.WindowEvent, *sdl.TouchFingerEvent, *sdl.CommonEvent:
+	case *sdl.MouseMotionEvent, *sdl.WindowEvent, *sdl.TouchFingerEvent, *sdl.CommonEvent, *sdl.KeyboardEvent, *sdl.TextInputEvent:
```
Comment (comtom) [inline scene.go:87]: And these removes some messages from stdout so It's not that verbose.

**Review by comtom — COMMENTED** [review]
Just a few notes, to clarify this PR.

**Comment by nieup** [comment]
👍

**Pull request closed without merging**